src/main/java/com/cognifide/knotx/handler/RestServiceResponseHandler.java

```diff
@@ -70,19 +66,25 @@ public void handle(HttpClientResponse response) {
         response.bodyHandler(buffer -> {
             String responseContent = buffer.getString(0, buffer.length());
             LOGGER.debug("Request in: " + request.absoluteURI() + " for " + dataCallUri);
+            Map serviceData = new Gson().fromJson(responseContent, Map.class);
```

Comment:
You can avoid using the raw `Map` type by specifying a type token as explained in the [GSON FAQ](https://sites.google.com/site/gson/gson-user-guide#TOC-Serializing-and-Deserializing-Generic-Types)